### ISSUE TITLE
Diagonal line character U+2571 is too small

### ISSUE THREAD

**Issue by melkster** [tonsky/FiraCode#1414]
In most fonts the character `╱` ([`U+2571`](https://unicode-table.com/en/2571/) ) spans from top to bottom which allows drawing diagonal lines over multiple lines, however in FiraCode it doesn't.

This is what it looks like in most other fonts (this is NotoSansMono Nerd Font):

![image](https://user-images.githubusercontent.com/15816726/169615492-e4fdfb80-48be-4628-98b8-661f63451e2e.png)

This is what it looks like in FiraCode:

![image](https://user-images.githubusercontent.com/15816726/169615468-376ca2f1-be23-489b-ad0d-2e32c137e028.png)

**Comment by tonsky** [comment]
Not really. This is what they look in Fira Code (the empty space in the middle is a bug, will fix)

<img width="788" alt="Screenshot 2022-05-22 at 22 47 52" src="https://user-images.githubusercontent.com/285292/169715201-adf09dda-30ac-4d0b-8367-8fad18cf7829.png">

**Comment by tonsky** [comment]
With the fix (in v7):

<img width="827" alt="Screenshot 2022-05-22 at 22 51 29" src="https://user-images.githubusercontent.com/285292/169715436-0f8d913e-1aa9-4a6c-b6dc-eaca45737838.png">

**Comment by melkster** [comment]
You're probably right. I realize that I'm running a version of FiraCode that I patched like a year ago or so. I'll make sure to update. Thanks!
